src/lib/register-service-worker.js

```diff
@@ -98,14 +98,25 @@ const startRegistration = (swUrl) => {
         });
 }
 
+export function sanitizedUrlGenerator (releaseHash: string, serviceWorker: string): string {
```

Comment:
nit: could we prefix this with `get` to make it clear that it returns a string?
```suggestion
export function getSanitizedUrl (releaseHash: string, serviceWorker: string): string {
```